### ISSUE TITLE
Wrong indentation after break between curly braces

### ISSUE THREAD

**Issue by NekR** [Microsoft/TypeScript-Sublime-Plugin#199]
Problem in action:
![2015-05-19-1210-42](https://cloud.githubusercontent.com/assets/1189265/7699556/652b749a-fe21-11e4-9222-1970be2ad29f.gif)

My config is:
```json
{
	"Seti_ClosedFolder_dots": true,
	"Seti_sb_small_padding": true,
	"Seti_sb_tree_miny": true,
	"Seti_tabs_small": true,
	"caret_extra_width": 2,
	"color_scheme": "Packages/Seti_UI/Scheme/Seti_orig.tmTheme",
	"draw_white_space": "all",
	"fade_fold_buttons": false,
	"font_size": 12.0,
	"highlight_line": true,
	"highlight_modified_tabs": true,
	"ignored_packages":
	[
		"Vintage"
	],
	"indent_guide_options":
	[
		"draw_active"
	],
	"line_padding_bottom": 1,
	"line_padding_top": 0,
	"minimap_scroll_to_clicked_text": false,
	"rulers":
	[
		80,
		100,
		120
	],
	"save_on_focus_lost": true,
	"shift_tab_unindent": true,
	"spell_check": true,
	"tab_size": 2,
	"theme": "Seti.sublime-theme",
	"translate_tabs_to_spaces": true,
	"typescript_auto_format": false,
	"typescript_auto_indent": false,
	"word_wrap": true
}
```

I am not sure right now if is happens only for ```{}```, but it's really annoying as were mentioned in https://github.com/Microsoft/TypeScript-Sublime-Plugin/issues/158#issuecomment-103609334

**Comment by zhengbli** [comment]
It turns that JavaScript has a special "JavaScript Indent.tmPreference" file that handles the mentioned indentation cases. Added to TypeScript also in PR #200.

**Comment by NekR** [comment]
@zhengbli nice, thank you! Looking forward to test it.

**Comment by zhengbli** [comment]
Fixed by #200
